src/services/edusharing/hooks/player.hooks.js

```diff
@@ -0,0 +1,33 @@
+const { Configuration } = require('@hpi-schul-cloud/commons');
+const { authenticate } = require('@feathersjs/authentication');
+const { disallow } = require('feathers-hooks-common');
+const { hasPermission, hasSchoolPermission } = require('../../../hooks');
+const { NotFound } = require('@feathersjs/errors');
+
+const isEdusharing = (context) => {
+	if (Configuration.get('FEATURE_LERNSTORE_ENABLED') !== true) {
+		throw new NotFound('This feature is disabled.');
```

Comment:
Can you please look if you find a better matching error type instead?

Reply:
Change the error type to 'MethodNotAllowed' in new commit.